### PR TITLE
BD-2651: Negative purchases (refunds) impact on LTV

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2651**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md [hunk 1]
```diff
@@ -29,47 +27,112 @@ The following lists methods across various platforms that are used to log purcha
...
 
-![Purchase graph on the Revenue page showing statistics for all purchases][4]
+##### Impact of refunds on lifetime revenue
 
-![Purchase breakdown table on the Revenue page listing the products in your applications, the number of times they've been purchased, and their associated revenue][3]
+When using purchase events to track purchase data, you should track refunds by logging a Braze purchase event with a negative `price` property. This approach will maintain an accurate total for the lifetime revenue.
+
+However, keep in mind the refund will count as an additional purchase event. Let's consider the following example. Sam makes their first purchase for $12 but returns part of the purchase for a refund of $5. Sam's profile would log:
+
+- Purchase, price $12
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:131]: Should this be
```suggestion
- **Purchase price:** $12
```

Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:131]: Should "Purchase" be bolded? Or should the formatting be different from **Lifetime revenue:**?

**Review by rachel-feinberg — APPROVED** [review]
Left a small comment about formatting, but otherwise looks good to me!

File: _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md [hunk 2]
```diff
@@ -29,47 +27,112 @@ The following lists methods across various platforms that are used to log purcha
...
+- Purchase, price -$5
+- **Lifetime revenue:** $7
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:133]: @rachel-feinberg for your suggestion on formatting, is this better? 
```suggestion
- 1 purchase with a price of $12
- 1 purchase with a price of -$5
- Lifetime revenue of $7
```

Comment (rachel-feinberg) [reply _docs/_user_guide/data_and_analytics/custom_data/purchase_events.md:133]: @bre-fitzgerald Yeah! I like the consistency here

**Pull request merged**